### DISCUSSION TITLE
[issue] Multiselect combobox : not able to select recently de-selected items from formpill group

### DISCUSSION THREAD

**Discussion by Joshi-Kajal** [twilio-labs/paste#2115] [Q&A] [answered]
Hey Team, 

I have integrated paste multiselect combobox in my project (SSO) and found one issue.

Issue is :  once we select any item from combobox and remove it from selected items (formpill group), user can not immediately add/select it back.

**Steps to reproduce :**
- Select item "X" from combobox
- remove item "X" from selected item list 
- item "X" will be again available in combobox list to select
- immediately again try to select item "X" from list

Expected Result: 
item "X" should be selected and visible as selected item below combobox as pill

Actual Result:
Upon selecting item "X", it is not added in selected item list below combobox

I am attaching video to make it more clear: 

https://user-images.githubusercontent.com/96221750/148387393-50f10e27-71c8-4c7c-90fa-eefd9ac42469.mov

**Comment by shleewhite** [comment]
Hi @Joshi-Kajal, thanks for letting us know about this bug! We are working on a fix now: https://github.com/twilio-labs/paste/pull/2119

**Comment by Joshi-Kajal** [comment]
Hey @shleewhite , when can we expect this change to be available?

**Reply by SiTaggart** [reply]
Hi, it won't necessarily be a "fix" more of an update to the example we show. The PR is here https://github.com/twilio-labs/paste/pull/2119

It's an implementation change

**Comment by gloriliale** [comment, marked as answer]
Hi @Joshi-Kajal! The implementation has been updated and can be [seen on our docs](https://paste.twilio.design/primitives/combobox-primitive#multiselect-combobox-example). Thanks for pointing this out!
